supabase/table_supabase_organization.go

```diff
@@ -0,0 +1,51 @@
+package supabase
+
+import (
+	"context"
+
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+)
+
+//// TABLE DEFINITION
+
+func tableSupabaseOrganization(ctx context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "supabase_organization",
+		Description: "Supabase Organization",
+		List: &plugin.ListConfig{
+			Hydrate: listSupabaseOrganizations,
+		},
+		Columns: []*plugin.Column{
+			{Name: "name", Type: proto.ColumnType_STRING, Description: "The display name of the organization."},
+			{Name: "id", Type: proto.ColumnType_STRING, Description: "A unique identifier of the organization."},
+		},
+	}
+}
+
+//// LIST FUNCTION
+
+func listSupabaseOrganizations(ctx context.Context, d *plugin.QueryData, _ *plugin.HydrateData) (interface{}, error) {
```

Comment:
With the authentication method made available, will the permission be enough to query the organization table?

Reply:
@misraved The access to the APIs depends on the user's access since the APIs are authenticated using a personal access token.
https://supabase.com/docs/reference/api/start#authentication